### PR TITLE
translation "Hindi" updated

### PR DESCRIPTION
**Pull request by Padmanabh82**

### CODE REVIEW

File: src/components/translation/TranslationList.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ const TranslationList = () => {
       icon: '🇫🇷'
     },
     {
-      lang: 'hi',
+      lang: 'hin-IND',
```
Comment (froxcey) [inline src/components/translation/TranslationList.tsx:48]: Did you read the translation guideline?

**Review by froxcey — CHANGES_REQUESTED** [review]
hin-IND is not a valid localisation name

File: src/components/translation/TranslationList.tsx (hunk 1 above)
Comment (Padmanabh82) [reply src/components/translation/TranslationList.tsx:48]: "Hin" stands for hindi and "IND" stands for india (mostly everywhere) but i as per ["this page"](https://docs.oracle.com/cd/E23824_01/html/E26033/glset.html) its hi-IN. Donno its matter or not but should i change it ?

Comment (Padmanabh82) [reply src/components/translation/TranslationList.tsx:48]: Okay

**Comment by froxcey** [comment]
Your PR were made when the translation library were getting an update to use YAML format. I see your changes, but merging it means I need to PR fixes for the incompatibilities. I have re-applied every changes in #170 and it will apply once it gets reviewed and merged. Thank you for your contribution, and closing this now.

**Comment by Padmanabh82** [comment]
#171

**Pull request closed without merging**